coupler/ensemble_manager.F90

```diff
@@ -409,8 +408,7 @@ subroutine ensemble_pelist_setup(concurrent, atmos_npes, ocean_npes, land_npes,
        !Append ensemble_id to the restart filenames
 
        !< Both calls are needed for cases where both fms2io/fmsio are used
-       call fms2_io_set_filename_appendix(trim(text))
-       call fms_io_set_filename_appendix(trim(text))
+       call set_filename_appendix(trim(text))
```

Comment:
Both of these calls are needed. It will break models that are still using fms_io.

Reply:
I put these back in